### ISSUE TITLE
Error starting Satisfactory with latest build/version

### ISSUE THREAD

**Issue by OHB** [Panakotta00/FicsIt-Networks#237]
Hello!

I haven't been in Satisfactory in about 10 days so when I opened Epic it updated to the latest version. I'm on the Early Release branch. Mod manager updated including an update to 0.3.11 of Ficsit Networks. 
![image](https://user-images.githubusercontent.com/3384372/203640667-3afa30ce-ec56-4c10-a3bb-55ec9bb0b151.png)

I've verified files, disabled/enabled mods, downgraded to 0.3.10 all to no avail. If there is additional information or logs I can provide I'm more than happy to.

Thanks for such a great plugin!

**Comment by OHB** [comment]
Found some relevant info:

[2022.11.23-20.30.24:348][  0]LogWindows: Failed to load 'C:/Program Files/Epic Games/SatisfactoryEarlyAccess/FactoryGame/Mods/FicsItNetworks/Binaries/Win64/FactoryGame-FicsItNetworks-Win64-Shipping.dll' (GetLastError=127)
[2022.11.23-20.30.24:348][  0]LogModuleManager: Warning: ModuleManager: Unable to load module 'C:/Program Files/Epic Games/SatisfactoryEarlyAccess/FactoryGame/Mods/FicsItNetworks/Binaries/Win64/FactoryGame-FicsItNetworks-Win64-Shipping.dll' because the file couldn't be loaded by the OS.
[2022.11.23-20.30.25:973][  0]Message dialog closed, result: Ok, title: Message, text: Plugin 'FicsItNetworks' failed to load because module 'FicsItNetworks' could not be loaded.  There may be an operating system error or the module may not be properly set up.

I checked. The file exists and is not empty. Hope that helps!
